Library/Homebrew/requirements/macos_requirement.rb

```diff
@@ -5,32 +5,58 @@
 class MacOSRequirement < Requirement
   fatal true
 
-  def initialize(tags = [])
-    @version = MacOS::Version.from_symbol(tags.shift) unless tags.empty?
+  def initialize(tags = [], comparator: ">=")
```

Comment:
Thoughts on making `comparator` a symbol like `:maximum` or `:greater_than` instead of a string?

Reply:
I think that would just introduce more boilerplate.

Reply:
🆒